modules/openxr/doc_classes/OpenXRInterface.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="OpenXRInterface" inherits="XRInterface" version="4.0">
+	<brief_description>
+		Our OpenXR interface.
+	</brief_description>
+	<description>
+		The OpenXR interface allows Godot to interact with OpenXR runtimes and make it possible to create XR experiences and games.
+		Due to the needs of OpenXR this interface works slightly different then other plugin based XR interfaces. It needs to be initialised when Godot starts. You need to enable OpenXR, settings for this can be found in your games project settings under the XR heading. You do need to mark a viewport for use with XR in order for Godot to know which render result should be output to the headset.   
+	</description>
+	<tutorials>
+		<link title="OpenXR documentation">$DOCS_URL/tutorials/vr/openxr/index.html</link>
```

Comment:
I've added the link to the tutorial here, on the stable docs this points to the Godot 3 documentation, on latest its missing, all the more reason to make sure this page gets written soon :)